FieldOpt/ERTWrapper/eclgridreader.cpp

```diff
@@ -135,39 +136,41 @@ ECLGridReader::Dims ECLGridReader::Dimensions()
 
 int ECLGridReader::ActiveCells()
 {
-    if (ecl_grid_ == 0) throw GridNotReadException("Grid must be read before gettign active cells.");
+    if (ecl_grid_ == 0) throw GridNotReadException("Grid must be read before getting the number of active cells.");
     else return ecl_grid_get_nactive(ecl_grid_);
 }
 
+bool ECLGridReader::IsCellActive(int global_index)
+{
+    if (ecl_grid_ == 0) throw GridNotReadException("Grid must be read before getting the active status of cells.");
+    else return ecl_grid_cell_active1(ecl_grid_, global_index);
+}
+
 ECLGridReader::Cell ECLGridReader::GetGridCell(int global_index)
 {
     if (!GlobalIndexIsInsideGrid(global_index))
-        throw InvalidIndexException("The global index " + boost::lexical_cast<std::string>(global_index) + " is outside the grid.");
+        throw InvalidIndexException("The global index " + std::to_string(global_index) + " is outside the grid.");
     if (ecl_grid_ == 0) throw GridNotReadException("Grid must be read before getting grid cells.");
```

Comment:
EB: didn't you have some issues with to_string on maur, which is why you shifted over to boost's lexical_cast?